tests/marketplace/deployer_helm_tiller_base/onbuild/standard/schema.yaml

```diff
@@ -12,6 +12,9 @@ properties:
     type: string
     x-google-marketplace:
       type: APPLICATION_UID
+      applicationUid:
+        generatedProperties:
+          createApplicationBoolean: global.application.create
```

Comment:
Can we change the `helm-dependency-build` variant of this test, too?

https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/tree/541156e4055c4940b205f732fbdf95c5c28b2dff/tests/marketplace/deployer_helm_tiller_base/onbuild/helm-dependency-build

Reply:
Was thinking that we should use both variants in test. It's not a very systematic testing approaching (as the tests are intended to test different unrelated aspects), but it makes me feel better for now.

Let me know if you feel strongly. I can follow up.